lessonbuilder/tool/src/resources/messages.properties

```diff
@@ -501,7 +501,7 @@ simplepage.permissions-general=You do not have the required permissions to do th
 simplepage.comment-unknown-user=Unknown User
 simplepage.comment-you=You
 simplepage.comment-anonymous-message=Normal users can only see the Anonymous names. The commenters' actual names are shown only to you and other site administrators.
-simplepage.edit-comment=Edit
```

Comment:
Can you revert these changes? You're changing the default locale.

Reply:
this was a double key

Reply:
good thanks